release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1391,41 +1392,45 @@ Starting in {product-title} 4.17, RukPak is now removed and relevant functionali
 //Telco Edge / TALO
 //Telco Edge / ZTP
 
+////
 [discrete]
 [id="ocp-4-17-api-auth-bug-fixes_{context}"]
 ==== API Server and Authentication
+////
 
 [discrete]
 [id="ocp-4-17-bare-metal-hardware-bug-fixes_{context}"]
 ==== Bare Metal Hardware Provisioning
 
-* Previously, attempting to configure RAID on specific hardware models by using Redfish might have resulted in the following error: "The attribute StorageControllers/Name is missing from the resource." With this update, the validation logic no longer requires the `Name` field, because it is not mandated by the Redfish standard. (link:https://issues.redhat.com/browse/OCPBUGS-38465[*OCPBUGS-38465*])
+* Previously, attempting to configure RAID on specific hardware models by using Redfish might have resulted in the following error: `The attribute StorageControllers/Name is missing from the resource`. With this update, the validation logic no longer requires the `Name` field, because it is not mandated by the Redfish standard. (link:https://issues.redhat.com/browse/OCPBUGS-38465[*OCPBUGS-38465*])
```

Comment:
```suggestion
* Previously, attempting to configure RAID on specific hardware models by using Redfish might have resulted in the following error: `The attribute StorageControllers/Name is missing from the resource`. With this update, the validation logic no longer requires the `Name` field because the field is not mandated by the Redfish standard. (link:https://issues.redhat.com/browse/OCPBUGS-38465[*OCPBUGS-38465*])
```